### ISSUE TITLE
Fix ESLint script

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#503]
@eileenchoe reported that the linting script that @NAnguiano wrote is broken now, so it needs to be fixed so that it can be run on the new merged code from PR #502.

**Comment by mihirsamdarshi** [comment]
Been working on this for a bit, and I was able to get it to lint, however I am now getting this error in my console every time I run it, even though the rest of the console is the proper result of the linter.

[`npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "lint"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! grnsight-beta@2.1.5 lint: `eslint **/*.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the grnsight-beta@2.1.5 lint script 'eslint **/*.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the grnsight-beta package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint **/*.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs grnsight-beta
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls grnsight-beta
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mihir/Documents/GRNsight/npm-debug.log`]

**Comment by mihirsamdarshi** [comment]
ESLint is fixed. The preferred command is `eslint **/*` to lint all the files, or to lint certain files, navigate into them and run `eslint filename`

**Comment by kdahlquist** [comment]
This will become part of @mihirsamdarshi's next pull request.

**Comment by kdahlquist** [comment]
Assigning this review to you, @dondi.

**Comment by dondi** [comment]
This works as expected. I also reviewed the style rules and made some tweaks. I pushed these directly to `beta`; if you look at the recent commits from me, @eileenchoe and @mihirsamdarshi you will be able to tell what rules I added. (in a nutshell, always have a space between `function` and `(`; disallow spaces between a function name and the `(` in a function call; no trailing spaces at the end of a line)
